src/core/annotation.js

```diff
@@ -1088,6 +1088,8 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
         break;
       }
     }
+    this.checkedAppearance = normalAppearanceState.get(this.data.buttonValue);
+    this.uncheckedAppearance = normalAppearanceState.get("Off") || null;
```

Comment:
Hm, this was already there, but it might be better to rename `normalAppearanceState` to `normalAppearance` here too like we did for the checkboxes to match the specification more closely.